packages/lib/src/settings/options/shouldIndexDirectShareLinkInSEO.js

```diff
@@ -0,0 +1,14 @@
+import { INPUT_TYPES } from '../utils/constants';
+import { default as GALLERY_CONSTS } from '../../common/constants';
+
+export default {
+  title: 'Search engines indexing for direct share links',
+  description: `Choose whether you want the items with share link to be indexed by crawlers in SEO or not`,
+  isRelevant: (sp) =>
```

Comment:
This is just my taste but I cna 
```[GALLERY_CONSTS.itemClick.FULLSCREEN, GALLERY_CONSTS.itemClick.EXPAND].includes(sp.itemClick)
```

Reply:
not sure I understand here, I see an empty line

Reply:
oops my bad. I would do (but it's just a matter of style)
```
[GALLERY_CONSTS.itemClick.FULLSCREEN, GALLERY_CONSTS.itemClick.EXPAND].includes(sp.itemClick)
```
You gain two things:
1. You don't have a duplication of `sp.itemClick ===`
2. You can easily add more values to the array in the future

Reply:
cool, I'll do that